### PR TITLE
Features at a glance

### LINKED ISSUE #180: Add features at a glance section

### PR DESCRIPTION
**Pull request by stopthatcow**
Add features at a glance to readme

Fixes #180

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -24,8 +24,19 @@ Getting Started
...
+Features at a glance
+--------------------
+
+- Branch creation from new or existing tickets.
+- Issue creation to associate branch with issue.
+- Code review creation to associate review with issue.
+- Code style enforcement/fix on commit (via git hook).
+- Inserts branch name into every commit (via git hook).
+- Open code review and issue webpages from terminal.
+- `git-flow <https://nvie.com/posts/a-successful-git-branching-model/>`__ branching model.
```
Comment (aramnhammer) [inline README.rst]: This is optional right ? user can specify the base branch.

File: test.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    url='https://zazucli.atlassian.net/',
+    username='nhwiles@gmail.com',
+    password='ZdgU1Xtft8OoQu1ZFNu085D6')
+
+me = j.issue('ZZ-5', fields='assignee')
+print(me['key'])
+try:
+    j.assign_issue(me['key'], 'asdf')
+except Exception as e:
+    print(e)
```
Comment (aramnhammer) [inline test.py]: not sure if `tox` calls `pytest` with the `-s` command but it might be necessary to see any print statements like this.

**Review by aramnhammer — APPROVED** [review]

File: README.rst (hunk 1 above)
Comment (stopthatcow) [reply README.rst]: Yes, that's true. But so is the git flow tool...I can just say "configurable"

File: test.py (hunk 2 above)
Comment (stopthatcow) [reply test.py]: This file was not meant to be part of this change :)

**Pull request merged**